README.md

```diff
@@ -1,37 +1,53 @@
 # Wormhole Solidity SDK
 
-The purpose of this SDK is to provide helpers to take your existing single-chain solidity application cross-chain using Wormhole's automatic relayers
+The purpose of this SDK is to make integrations with Wormhole on EVM compatible chains as smooth as possible by providing all necessary Solidity interfaces along with useful libraries and tools for testing.
 
-### Installation
+This SDK was originally created for integrations with the WormholeRelayer and then expanded to cover all integrations.
+
+## Installation
 
 **Foundry and Forge**
 
 ```bash
 forge install wormhole-foundation/wormhole-solidity-sdk
 ```
 
-### Example Usage + Introduction to Automatic Relayers
+**Solc Version**
+
+Currently the SDK uses solc version 0.8.19 to avoid issues with OPCODE0 which was introduced in 0.8.20 but which is not supported on many EVM chains.
```

Comment:
```suggestion
Currently the SDK uses solc version 0.8.19 to avoid issues with PUSH0 which was introduced in 0.8.20 but which is not supported on many EVM chains.
```

Reply:
should we use more permissive compiler version bounds (in the source files) and suggest instead to target `paris`?

Reply:
Yes, I [had to do that in the branch where I added openzeppelin](https://github.com/wormhole-foundation/wormhole-solidity-sdk/blob/9c3852e5ff83f6e5c6ec969c2ac5cd97b163eede/foundry.toml) because it is on 8.20 anyway...

Reply:
Strictly speaking, one should be going over individual files to determine their necessary pragma versions... some make use of solc features that were only introduced in the high teens while others are entirely unconstrained.